migrations/0044_sub_issuer_creds.up.sql

```diff
@@ -0,0 +1,3 @@
+-- adding metadata about the credentials (used first for validity period)
+alter table order_creds add column metadata jsonb default null;
```

Comment:
Is there a reason you can't split these out?

Reply:
These schema changes are required for the skus/datastore.go method additions.  We are going to store the validTo and validFrom in the metadata.